criu/config.c

```diff
@@ -832,6 +862,17 @@ int parse_options(int argc, char **argv, bool *usage_error,
 				return 1;
 			}
 			break;
+		case 1098:
+			if (parse_file_validation_method(&opts, optarg))
+				return 2;
+			break;
+		case 1099:
+			opts.file_validation_chksm_parameter = atoi(optarg);
+			/* The checksum parameter has to be less then 10MB for performance reasons. */
+			if (opts.file_validation_chksm_parameter <= 0 ||
+					opts.file_validation_chksm_parameter > 10485760)
```

Comment:
I don't think that we need this limit. We need to have a reasonable default value.

Reply:
I've removed this check, the default value is 1024.